.travis.yml

```diff
@@ -0,0 +1,16 @@
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -qq openssh-server
+  - mkdir -p ~/.ssh/
+  - chmod 700 ~/.ssh/
```

Comment:
```
chmod u+rwx,go= ~/.ssh
```
Not sure if it is better, but for me personally it is easier to read — not everyone knows about their meaning.